x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_exploration/components/regression_exploration/regression_exploration.tsx

```diff
@@ -4,25 +4,61 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React, { FC, Fragment } from 'react';
-// import { i18n } from '@kbn/i18n';
-import { EuiSpacer } from '@elastic/eui';
-
+import React, { FC, Fragment, useState, useEffect } from 'react';
+import { EuiSpacer, EuiLoadingSpinner, EuiPanel } from '@elastic/eui';
+import { ml } from '../../../../../services/ml_api_service';
+import { DataFrameAnalyticsConfig } from '../../../../common';
 import { EvaluatePanel } from './evaluate_panel';
-// import { ResultsTable } from './results_table';
+import { ResultsTable } from './results_table';
+import { DATA_FRAME_TASK_STATE } from '../../../analytics_management/components/analytics_list/common';
+
+interface GetDataFrameAnalyticsResponse {
+  count: number;
+  data_frame_analytics: DataFrameAnalyticsConfig[];
+}
+
+const LoadingPanel: FC = () => (
+  <EuiPanel>
+    <EuiLoadingSpinner className="mlRegressionExploration__evaluateLoadingSpinner" size="xl" />
+  </EuiPanel>
+);
 
 interface Props {
   jobId: string;
-  destIndex: string;
-  dependentVariable: string;
+  jobStatus: DATA_FRAME_TASK_STATE;
 }
 
-export const RegressionExploration: FC<Props> = ({ jobId, destIndex, dependentVariable }) => {
+export const RegressionExploration: FC<Props> = ({ jobId, jobStatus }) => {
+  const [jobConfig, setJobConfig] = useState<DataFrameAnalyticsConfig | undefined>(undefined);
+  const [isLoadingJobConfig, setIsLoadingJobConfig] = useState<boolean>(false);
+
+  useEffect(() => {
+    (async function() {
+      setIsLoadingJobConfig(true);
+      const analyticsConfigs: GetDataFrameAnalyticsResponse = await ml.dataFrameAnalytics.getDataFrameAnalytics(
```

Comment:
This will fail silently, suggest to add a `try/catch` block and display a toast or callout for errors.